src/test/cypress/e2e/exam/ExamParticipation.cy.ts

```diff
@@ -5,13 +5,14 @@ import submission from '../../fixtures/exercise/programming/all_successful/submi
 import { Course } from 'app/entities/course.model';
 import { generateUUID } from '../../support/utils';
 import { EXERCISE_TYPE } from '../../support/constants';
-import { courseManagementRequest, examExerciseGroupCreation, examNavigation, examParticipation, examStartEnd } from '../../support/artemis';
-import { Exercise } from 'src/test/cypress/support/pageobjects/exam/ExamParticipation';
+import { courseManagementRequest, examExerciseGroupCreation, examNavigation, examParticipation, examStartEnd, textExerciseEditor } from '../../support/artemis';
+import { AdditionalData, Exercise } from 'src/test/cypress/support/pageobjects/exam/ExamParticipation';
```

Comment:
Is this client style issue correct? Do we need this import?

Reply:
Ah good point, I think that is a left over from the rebase. Thanks for the hint!